dw-dialog.js

```diff
@@ -381,6 +383,17 @@ export const DwModalDialogMixin = (baseElement) => class DwModalDialog extends b
     this.dispatchEvent(event);
     this._unlistenEvents();
 
+    const index = window.openedDwDialogsInstances.findIndex((element) => element === this);
+    if (index >= 0) {
+      window.openedDwDialogsInstances.splice(index, 1);
```

Comment:
@devangdreamworld I think we always need to remove last element. So, we don't need to find index first and then remove that element from list. Instead just remove last element directly.